vro-streamlit/src/dev-requirements.txt

```diff
@@ -5,4 +5,6 @@ pytest>=7.4
 pytest-cov>=5.0
 pytest-env>=1.0
 pytest-mock>=3.11
+requests-mock>=1.10
 ruff>=0.6
+types-requests>=2.32
```

Comment:
What are your thoughts on version pining? Too soon?
`requests>=2.32,<3.0`

Reply:
Yes we should pin it!